src/components/zen-badge/zen-badge.scss

```
@@ -0,0 +1,16 @@
+@import '../scss/component-helpers.scss';
+
+:host {
+  display: block;
+  background: $color-gray-100;
+  border-radius: 1rem;
+  margin-left: 0.5rem;
+  padding: 0 0.5rem;
+  min-width: 1rem;
+  text-align: center;
```

nit: Would be nice to also center it vertically in case we have some icons or something else in default slot. It can be overridden by consumer, but default value should be the most useful one:
```suggestion
  display: flex;
  align-items: center;
  justify-content: center;
```

---

Addded